### ISSUE TITLE
Inconsistent default text properties

### ISSUE THREAD

**Issue by gagern** [CindyJS/CindyJS#49]
Geometrix labels and `drawtext` use different font families and different text sizes by default. This is somewhat counter-intuitive, and also in contrast to what Cinderella does.

Example, exported from Cinderella:

```js
createCindy({ 
  scripts: {draw: 'drawtext(A.xy + [1,-1], "B");'},
  defaultAppearance: {}, 
  geometry: [ 
    { name: "A", type: "Free", pos: [ 4.0, -1.804878048780488, -1.2195121951219514 ], color: [ 1.0, 0.0, 0.0 ], labeled: true } ], 
  ports: [{ id: "CSCanvas", width: 680, height: 336, transform: [ { visibleRect: [ -9.04, 9.32, 18.16, -4.12 ] } ], background: "rgb(168,176,192)" } ] });
```

CindyJS renders this as ![CindyJS screenshot](https://cloud.githubusercontent.com/assets/190759/10816839/2930f808-7e36-11e5-8e36-339c46316288.png) while Cinderella does ![Cinderella screenshot](https://cloud.githubusercontent.com/assets/190759/10816859/4fc3a8da-7e36-11e5-9ef0-13cd7fec5c18.png).

**Comment by kortenkamp** [comment]
Isn't the real problem that the Cinderella export does not export the properties completely?

**Comment by gagern** [comment]
@kortenkamp: I don't think that's the whole story. Different text-creating things using different defaults feels strange for those who write CindyJS code by hand, so there is definitely something to fix purely on the CindyJS side, in my opinion.

Cinderella might and probably should export text sizes for its labels explicitely, but it should not modify the text of the script to include an explicit `size ->` modifier which was not entered by the user. So `drawtext` should fit in with its surroundings so that the whole result looks about the same in Cinderella and in CindyJS. But since `drawtext` uses `csport.drawingstate.textsize` it should honour `defaultAppearance`, so setting that on export should fix things on export from Cinderella, even if CindyJS might choose to use a different font size by default.

As I just did look up the relevant places: `csport.drawingstate.textsize` [defaults to 20](https://github.com/CindyJS/CindyJS/blob/57a564d89124df6d86f605272bc91740fb2c2348/src/js/libgeo/GeoState.js#L14), while `el.textsize` used for the label of an element `el` [defaults to 12](https://github.com/CindyJS/CindyJS/blob/57a564d89124df6d86f605272bc91740fb2c2348/src/js/libgeo/GeoRender.js#L22). And apparently [`defaultAppearance`](https://github.com/CindyJS/CindyJS/blob/57a564d89124df6d86f605272bc91740fb2c2348/src/js/libgeo/GeoBasics.js#L1-L19) is something else again, with no immediately obvious connection to `csport.drawingstate`. Contrary to what I had assumed so far.

**Comment by gagern** [comment]
Not done yet: need to handle default text size.

**Comment by gagern** [comment]
Fixed by the combination of #146 and #345.
